README.md

```diff
@@ -12,23 +12,24 @@ approval for the full stack used through the Architecture and Security groups.
 ```rust
 let mauth_info = MAuthInfo::from_default_file().unwrap();
```

Comment:
I think this example should include the `use` statements or maybe we should have an `examples/main.rs` file with something that compiles and works

Reply:
Note that this is generated from [this doc test](https://github.com/mdsol/mauth-client-rust/pull/15/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L10-L32) which is compiled as part of the CI test suite.

Making an example that works is problematic, since that would require an actual MAuth URL and valid private key to be stored in this public repo. I suppose I can modify it to allow the `use` statements to be shown though.